packages/content-management/table_list/src/types.ts

```diff
@@ -12,3 +12,16 @@ export interface Tag {
   description: string;
   color: string;
 }
+
+export type TableRowAction = 'list' | 'delete';
```

Comment:
what is `list` for? is it used? As I understand we filter server-side outside of table_list.

Reply:
Good catch. It's a leftover. Removed in https://github.com/elastic/kibana/pull/155179/commits/ac2153b29d43c2f45d914dbdfc8f089715a29d51